analyses/Linelist_cs_ebola_simulations.R

```diff
@@ -0,0 +1,278 @@
+
+# Script for individual-level data case study 4: disease with a delay from onset
+# to death shorter than onset to recovery (Ebola-like outbreak).
+# Methods to compare:
+# 1) Naive estimate of deaths/cases
+# 2) Retrospective cohort by omitting latest cases for which we don't expect to have an outcome (i.e., within 1 onset-death before real-time)
+# 3) Using {cfr} function estimate_outcomes to recalculate denominator
+# 4) Using {cfr} function cfr_static
+
+
+# Using simulist to simulate outbreak data
+# Columns that we need: onset date, column to indicate outcome, outcome date (recovery/death)
+
+library(data.table)
+library(zoo)
```

Comment:
Where are {data.table} and {zoo} used? It's personal preference, but you could namespace some functions (e.g. `pkgname::func()`).

Reply:
they were used previously but not in the final version, I'll remove them
and do you mean whenever I use a package to include the package name?